requirements/main.txt

```diff
@@ -1,3 +1,4 @@
 aiomysql
 prometheus_client
 everett[yaml]
+https://github.com/FAForever/faf-scfa-replay-parser/archive/0.1.zip
```

Comment:
Can't we just copy the code into this repo? I don't think replay parsing will gain another python dependency.

Reply:
I don't see what the downside of doing it this way is. What if someone wants to write another web based replay parsing app?

Reply:
In Python? Besides, having this code here lets us discard any processing we don't need.

Reply:
Flask and Django are quite popular AFAIK. 

It would be easier to push it to PyPi (this could even be automated in Travis), but I'm not sure what the implications of that would be.